### PR TITLE
Add basic docs

### PR DESCRIPTION
**Pull request by AnthonyLim23**
This PR adds the bare bones for documentation. It has some place holder text, but shows what we can do. 

To test:
go to the read the docs build and click details
You should then see some pages. 
Check they are ok - note that the auto generated text does not do nice equations :(

### CODE REVIEW

File: src/quickBayes/log_likelihood.py [hunk 1]
```diff
@@ -37,6 +33,7 @@ def loglikelihood(x_len: ndarray, chi2: float, covar: ndarray,
     :return the loglikelihood
     """
 
+    log_hess_det = log10_hessian_det(covar)
```
Comment (AnthonyLim23) [inline src/quickBayes/log_likelihood.py:36]:
```suggestion
```

File: src/quickBayes/log_likelihood.py [hunk 2]
```diff
@@ -48,7 +45,6 @@ def loglikelihood(x_len: ndarray, chi2: float, covar: ndarray,
     We will therefore add a penality to the loglikelihood
     via the hessian.
     """
-    log_hess_det = log10_hessian_det(covar)
     if np.max(np.abs(covar)) > 1:
```
Comment (AnthonyLim23) [inline src/quickBayes/log_likelihood.py:48]:
```suggestion
log_hess_det = log10_hessian_det(covar)
    if np.max(np.abs(covar)) > 1:
```

File: src/quickBayes/log_likelihood.py [hunk 3]
```diff
@@ -48,7 +44,7 @@ def loglikelihood(x_len: ndarray, chi2: float, covar: ndarray,
     We will therefore add a penality to the loglikelihood
     via the hessian.
     """
-    log_hess_det = log10_hessian_det(covar)
+log_hess_det = log10_hessian_det(covar)
```
Comment (AnthonyLim23) [inline src/quickBayes/log_likelihood.py:47]:
```suggestion
    log_hess_det = log10_hessian_det(covar)
```

File: docs/source/loglikelihood.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+=============
+
+Calculate the unnormalised logliklihood.
+log_10 probabilities -> priors
+The equation for the probability is taken from "Data Analysis A bayesian tutorial second edition", by D. S. Sivia equation 4.20 page 88:
+
+.. math::
+    P(M|D_k,I) \propto \frac{N! (4\pi)^N}{\beta^N\sqrt{\det(H)}}\exp{(-\frac{\chi^2}{2})},
+
+where :math:`\\beta = (x_{max}-x_{min})A_{max}`, :math:`H` is the Hessian matrixWe want this as a logliklihood, so take :math:`\\log_{10}` and use:
```
Comment (robertapplin) [inline docs/source/loglikelihood.rst:11]:
```suggestion
where :math:`\\beta = (x_{max}-x_{min})A_{max}`, :math:`H` is the Hessian matrix. We want this as a loglikelihood, so take :math:`\\log_{10}` and use:
```

**Review by robertapplin — CHANGES_REQUESTED** [review]
Looks good, I've just spotted a few typos then I'm happy to approve

**Review by robertapplin — APPROVED** [review]

**Pull request merged**